meta-balena-common/recipes-core/balena-rollback/files/rollback-parse-bootloader

```diff
@@ -55,8 +57,7 @@ fi
 # --uniq here because findmnt returns the same device twice because /mnt/sysroot/active
 # is bind-mounted over itself while running old hooks from the inactive partition
 current_part=$(findmnt --noheadings --canonicalize --uniq --output SOURCE /mnt/sysroot/active)
-type=$(lsblk -no TYPE "${current_part}")
-if [ "${type}" = "crypt" ]; then
+if is_part_encrypted "${current_part}"; then
```

Comment:
Does this behave the same for both LUKS and raw DM?

Reply:
Yes, `is_part_encrypted` can be used to detect both `dm-crypt` and LUKS partitions.